### ISSUE TITLE
Remove extra text in GPL license

### ISSUE THREAD

**Issue by ace-dent** [hadrien-psydk/pngoptimizer#13]
Consider removing extra GPL blurb from end of the license file:

https://github.com/hadrien-psydk/pngoptimizer/blob/e314eccefcfff8875c20285ff776b9465477f092/projects/pngoptimizercl/License.txt#L282

**Comment by hadrien-psydk** [comment]
Hello Andrew,

I have a doubt. I could not find any clue about the extra blurb removal:
  * I read the [how-to on gnu.org](https://www.gnu.org/licenses/gpl-howto.en.html) and it says nothing about that. It also points to a raw text file that does include the extra blurb.
  * I also looked at some other projects: linux, systemd... they have the extra blurb in their license file.
  * On my OS (Ubuntu) the extra blurb is there too, in /usr/share/common-licenses/GPL-2
  * I also created a new repository on GitHub and asked to initialize the repo with a GPL2 license. The extra blurb was there too.

Are you sure this extra blurb needs to be removed? What made you think this should be the case?
